doc/exception_interrupts.rst

```diff
@@ -94,13 +99,20 @@ Ibex can trigger an exception due to the following exception causes:
 The illegal instruction exception, instruction access fault, LSU error exceptions and ECALL instruction exceptions cannot be disabled and are always active.
 
 
-Handling
---------
+Nested Interrupt/Exception Handling
+-----------------------------------
 
-Ibex does support nested interrupt/exception handling.
-Exceptions inside interrupt/exception handlers cause another exception.
-However, exceptions during the critical part of your exception handlers, i.e. before having saved the ``mepc`` and ``mstatus``, will cause those CSRs to be overwritten.
-Interrupts during interrupt/exception handlers are thus disabled by default, but can be explicitly enabled if desired.
+Ibex does support nested interrupt/exception handling in software.
+However, interrupts/exceptions during the critical part of the handler, i.e. before software has saved the ``mepc`` and ``mstatus``, will cause those CSRs to be overwritten.
+The hardware thus automatically disables interrupts upon entering an interrupt/exception handler.
```

Comment:
Can you swap this sentence with the previous one? First explain what's happening, and then explain why it is implemented this way.

Reply:
Yep, good idea.